src/core/withExperiment.js

```diff
@@ -62,6 +62,7 @@ export type WithExperimentInjectedProps = {|
 type CookieConfig = {|
   maxAge?: number,
   path?: string,
+  secure?: boolean,
```

Comment:
We should probably have done that before, could you please add a comment with [this link](https://github.com/reactivestack/cookies/tree/f9beead40a6bebac475d9bf17c1da55418d26751/packages/react-cookie#setcookiename-value-options) above the type definition?